lib/bufbstore/buf_bstore.go

```diff
@@ -93,6 +107,25 @@ func (bs *BufferedBS) DeleteBlock(c cid.Cid) error {
 	return bs.write.DeleteBlock(c)
 }
 
+func (bs *BufferedBS) View(c cid.Cid, callback func([]byte) error) error {
+	if bs.writeviewer == nil || bs.readviewer == nil {
+		// none of the stores supports Viewer, just fall back to pure Get behaviour.
```

Comment:
comment is misleading, this happens when any of the stores doesn't support a viewer

Reply:
Oh yeah, leftover from an iteration.